html/php-pre-challenge2/index.php

```diff
@@ -3,7 +3,13 @@
 
 // 修正はここから
 for ($i = 0; $i < count($array); $i++) {
-
+    for ($j = 0; $j < count($array) - 1 - $i; $j++) { //比較する残りの値の数くりかえし
+        if ($array[$j] > $array[$j + 1]) { //右隣の値より大きければ
+            $order = $array[$j + 1];       //インデックスを+1した値を変数に代入しておく
```

Comment:
```suggestion
            $order = $array[$j + 1];
```

コードで書いていることと同じことを説明しているコメントは書かないようにしましょう。

* なぜ？やどうして？など、実装からは読み取れないこと
* 複雑でコメント書いておかないと理解できないこと
などに、コメントをつけます。